_posts/2022-07-27-#24858.md

```diff
@@ -10,15 +10,88 @@ status: upcoming
 commit: bcb0cac
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- A new node automatically initializes and populates its data directory by performing
+["Initial block download" (IBD)](https://bitcoin.stackexchange.com/a/106136/97099),
+which is the process of fetching blocks
+from peers and storing them in the `blocks` directory within the data directory
+(default `$HOME/.bitcoin`). 
+
+- The blocks are stored in files named `blknnnnn.dat` (for example, `blk01234.dat`).
+These files are limited to 128 MiB, so each can hold several hundred blocks.
```

Comment:
"Several hundred" seems too much, considering that the physical size of a block currently is 1.5-2MiB.

Reply:
I see what happened, the current block height is 746504, and my datadir has 3120 `blk*.dat` files, so that's 239 blocks per block file, which was the basis of my text. But you're right, with more recent post-segwit full blocks, only 64 blocks (at 2 MiB each) fit in one block file. I'll fix the text, thanks.